models/lung_nodule_ct_detection/configs/inference_raw.json

```diff
@@ -0,0 +1,55 @@
+{
+    "data_list_file_path": "$@bundle_root + '/LUNA16_datasplit/original/dataset_fold0.json'",
```

Comment:
If we use `"_disabled_"`, how to handle the change of `"data_list_file_path"` of the two setting?

Reply:
Actually, I feel we don't need to care about the `data_list_file_path`, if it's raw image, you need to set `resampled_image=True`.
Don't worry about the bundle APIs, I think @yiheng-wang-nv can help you on the development.

Thanks.

Reply:
But here, we provided download link for all the all the data list json files. I put json files for resampled and raw images in two different folders. It is already structured. That's why I put a `"data_list_file_path"` here.